tests/adapter/dbt/tests/adapter/concurrency/test_concurrency.py

```diff
@@ -0,0 +1,122 @@
+import pytest
+from pathlib import Path
+from dbt.tests.util import (
+    run_dbt,
+    check_relations_equal,
+    check_table_does_not_exist,
+    run_dbt_and_capture,
+)
+
+
+models__invalid_sql = """
```

Comment:
what is the significance of the double underscore a convention?

Reply:
@chenyu might have a greater understanding but yes a naming convention we use for these once applying the fixtures in same file, we later on alias them out to more user friendly names. my understanding( type of file/directory it lived in__name of file_then ref to if its yml or sql)

Reply:
@colin-rogers-dbt, @McKnight-42 has it right.  We use a script to convert old tests to new and this is how it handles naming them.  The main reason behind it is to prevent name collisions.  We often had this kind of structure as directories:

```
- models
  - valid.sql
  - others.sql
- some_other_models
  - valid.sql
  - others.sql
  - schema.yml
``` 

The new tests don't care about the actual name structure though.